lib/browser.js

```diff
@@ -0,0 +1,20 @@
+import { default as ClientInterface, types } from 'e3db-client-interface'
+import SodiumCrypto from './SodiumCrypto'
+
+export { default as helpers } from './helpers'
+
+const sodiumCrypto = new SodiumCrypto()
+/* eslint-disable-next-line new-cap */
+const Config = ClientInterface.Config
+/* eslint-disable-next-line new-cap */
+const Client = ClientInterface.Client(sodiumCrypto)
+const tozStoreTypes = types
+
+const tozStore = {
```

Comment:
After thinking about it, let's actually switch this to `tozny` from `tozStore`. This will likely contain other items. We may also want to use object extension rather than outright assignement, which is a jQuery-style lesson.

```
const tozny = {
// new object stuff here....
}
window.tozny = Object.assign( window.tozny || {}, tozny)
```

This is a really gentle way to add to the object and if we have other separate libraries down the line, it allows us to add to the global tozny namespace without worrying about load order.